src/store/model/newegg-ca.ts

```diff
@@ -417,6 +417,126 @@ export const NeweggCa: Store = {
 			url:
 				'https://www.newegg.ca/evga-geforce-rtx-3070-08g-p5-3753-kr/p/N82E16814487529'
 		},
+		{
+			brand: 'asus',
+			itemNumber: '14-126-471',
+			model: 'tuf',
+			series: '3060ti',
+			url:
+				'https://www.newegg.ca/asus-geforce-rtx-3060-ti-tuf-rtx3060ti-o8g-gaming/p/N82E16814126471'
+		},
+		{
+			brand: 'gigabyte',
+			itemNumber: '14-932-377',
+			model: 'gaming oc',
+			series: '3060ti',
+			url:
+				'https://www.newegg.ca/gigabyte-geforce-rtx-3060-ti-gv-n306tgaming-oc-8gd/p/N82E16814932377'
+		},
+		{
+			brand: 'gigabyte',
+			itemNumber: '14-932-375',
+			model: 'aorus',
+			series: '3060ti',
+			url:
+				'https://www.newegg.ca/gigabyte-geforce-rtx-3060-ti-gv-n306taorus-m-8gd/p/N82E16814932375'
+		},
+		{
+			brand: 'asus',
+			itemNumber: '14-126-468',
+			model: 'dual',
+			series: '3060ti',
+			url:
+				'https://www.newegg.ca/asus-geforce-rtx-3060-ti-dual-rtx3060ti-o8g/p/N82E16814126468'
+		},
+		{
+			brand: 'asus',
+			itemNumber: '14-126-470',
+			model: 'strix',
+			series: '3060ti',
+			url:
+				'https://www.newegg.ca/asus-geforce-rtx-3060-ti-rog-strix-rtx3060ti-o8g-gaming/p/N82E16814126470'
+		},
+		{
+			brand: 'evga',
+			itemNumber: '14-487-535',
+			model: 'xc gaming',
+			series: '3060ti',
+			url:
+				'https://www.newegg.ca/evga-geforce-rtx-3060-ti-08g-p5-3663-kr/p/N82E16814487535'
+		},
+		{
+			brand: 'gigabyte',
+			itemNumber: '14-932-376',
+			model: 'gaming oc pro',
+			series: '3060ti',
+			url:
+				'https://www.newegg.ca/gigabyte-geforce-rtx-3060-ti-gv-n306tgamingoc-pro-8gd/p/N82E16814932376'
+		},
+		{
+			brand: 'zotac',
+			itemNumber: '14-500-507',
+			model: 'gaming',
+			series: '3060ti',
+			url:
+				'https://www.newegg.ca/zotac-geforce-rtx-3060-ti-zt-a30610h-10m/p/N82E16814500507'
+		},
+		{
+			brand: 'evga',
+			itemNumber: '14-487-537',
+			model: 'ftw ultra',
```

Comment:
This is a `ftw3 ultra` card

Ref: https://www.evga.com/products/product.aspx?pn=08G-P5-3667-KR